var/spack/repos/builtin/packages/pgmath/package.py

```diff
@@ -29,9 +29,11 @@
 class Pgmath(CMakePackage):
     """Flang's math library"""
     homepage = "https://github.com/flang-compiler/flang"
-    url      = "https://github.com/flang-compiler/flang/archive/pgmath_20180319.tar.gz"
+    url      = "https://github.com/flang-compiler/flang/archive/flang_20180612.tar.gz"
```

Comment:
Should this package still be called `pgmath`?

Reply:
We have plans for pgmath to eventually be a standalone package used by others as well, not just for flang. The only other consideration for the name would be libpgmath but we've decided that pgmath is more appropriate.